modules/meeting/app/components/recurring_meetings/delete_dialog_component.html.erb

```diff
@@ -0,0 +1,46 @@
+<%#-- copyright
+OpenProject is an open source project management software.
+Copyright (C) the OpenProject GmbH
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License version 3.
+
+OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+Copyright (C) 2006-2013 Jean-Philippe Lang
+Copyright (C) 2010-2013 the ChiliProject Team
+
+This program is free software; you can redistribute it and/or
+modify it under the terms of the GNU General Public License
+as published by the Free Software Foundation; either version 2
+of the License, or (at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program; if not, write to the Free Software
+Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+See COPYRIGHT and LICENSE files for more details.
+
+++#%>
+
+<%= 
+  render(Primer::OpenProject::DangerConfirmationDialog.new(
+    id:,
+    title:,
+    form_arguments: {
+      action: polymorphic_path([@project, @meeting]),
```

Comment:
This was lifted from here: https://github.com/opf/openproject/pull/17569/files#diff-d10668a4a54d66743190f51acff89d1022705eeee43b368c366e7cabf564526aL42 However there is no project-scoped `destroy` route.

Reply:
I guess having only the global destroy makes sense, as long as you do a `redirect_back` as to not accidentally leave the project scope.

We have this issue in a few places in meeting, where some actions in a project context incorrectly redirect you to the global level. This results in you losing your project sidebar, etc.

Reply:
> I guess having only the global destroy makes sense, as long as you do a redirect_back as to not accidentally leave the project scope.

I ended up going with the opposite approach - have documented why in the PR description.